draft-ietf-scitt-architecture.md

```diff
@@ -614,103 +624,103 @@ The retrieved Receipt may be embedded in the corresponding COSE_Sign1 document i
+Nonetheless, Issuers should carefully review the inclusion of private/confidential materials in their issued Signed Statements; they may for instance remove any PII, or include instead opaque cryptographic commitments, such as hashes.
 
-**Queries to the Registry**
+#### Queries to the Registry
 
-The confidentiality of queries is implementation-specific, and generally not guaranteed. For example, while offline Claim verification is private, a TS may monitor which of its Claims are being verified from lookups to ensure their freshness.
+The confidentiality of queries is implementation-specific, and generally not guaranteed. For example, while offline Envelope validation of Signed Statements is private, a Transparency Services may monitor which of its Transparent Statements are being verified from lookups to ensure their freshness.
 
 ### Cryptographic Assumptions
 
-We rely on standard cryptographic security for signing schemes (EUF-CMA: for a given key, given the public key and any number of signed messages, the attacker cannot forge a valid signature for any other message) and for receipts schemes (log collision-resistance: for a given commitment such as a Merkle-tree root, there is a unique log such that any valid path authenticates a claim in this log.)
+We rely on standard cryptographic security for signing schemes (EUF-CMA: for a given key, given the public key and any number of signed messages, an attacker cannot forge a valid signature for any other message) and for Receipts schemes (log collision-resistance: for a given commitment such as a Merkle-tree root, there is a unique log such that any valid path authenticates a Signed Statement in this log.)
 
-SCITT supports cryptographic agility: the actors depend only on the subset of signing and receipt schemes they trust. This enables the gradual transition to stronger algorithms, including e.g. post-quantum signature algorithms.
+The SCITT Architecture supports cryptographic agility: the actors depend only on the subset of signing and Receipt schemes they trust. This enables the gradual transition to stronger algorithms, including e.g. post-quantum signature algorithms.
 
-### TS Clients
+### Transparency Service Clients
 
-Trust in clients that submit Claims for registration is implementation-specific. Hence, an attacker may attempt to register any Claim it has obtained, at any TS that accepts them, possibly multiple times and out of order. This may be mitigated by a TS that enforces restrictive access control and registration policies.
+Trust in clients that submit Signed Statements for registration is implementation-specific. Hence, an attacker may attempt to register any Signed Statement it has obtained, at any Transparency Service that accepts them, possibly multiple times and out of order. This may be mitigated by a Transparency Services that enforces restrictive access control and Registration policies.
```

Comment:
An untrusted Client could obtain legitimate credentials and masquerade as a legitimate user only to perform a denial of service attack in the form of out of order registrations and the like. The SCITT architecture should have some mechanism for ensuring operational integrity despite having untrusted Clients or it should require Client trust and employ some form of vetting process to register, authenticate and authorize them.

Reply:
Thanks @nedmsmith. The registration policy provides for a SCITT Transparency Service implementation to create various policies, including which identities, and or identity types are permitted. 
Note, there is a difference between a user that has RBAC writes to a ledger (Steve), and whether the signed statement from `the-ned-smith-project`, with an Issuer of `the-ned-smith-project` is accepted. 

The first barrier is RBAC control. And implementations will have various DOS configurations. Do you think this should be captured in the SCITT architecture, or be part of an implementation?

If there's additional clarity needed, can you please open a new issue?